docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -5,10 +5,16 @@ set -e
 cd /opt/coins/nodes/bitcoin_regtest
 
 echo "Starting bitcoin regtest backend service"
-/opt/coins/nodes/bitcoin_regtest/bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+
+sleep 3
+
+# generate test wallet and blocks
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc createwallet "tenv-wallet"
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc -generate 1024 1000
```

Comment:
I would say that `-generate 150` is enough here, we do not need so many BTC (1024 blocks will yield 14700 BTC, 150 is the "halving" interval, and we get 2500 spendable BTC from it - more than enough).

The second argument is also optional, based on [docs](https://developer.bitcoin.org/reference/rpc/generate.html), we do not need to include it (unless there is a reason for it - @vdovhanych ?)